Libraries/Motors/Motors.h

```diff
@@ -6,6 +6,7 @@
 #ifndef MOTORS_H
 #define MOTORS_H
 
+#include <Arduino.h>
```

Comment:
What's the difference between <Arduino.h> and "arduino.h"?

Reply:
It's the same, I did not realize I put it on twice.